HeRoBackEnd/Controllers/CandidateController.cs

```diff
@@ -46,45 +63,79 @@ public async Task<IActionResult> Get(int? id)
         //    return new JsonResult(new Candidate());
         //}
 
+        /// <summary>
+        /// Creates a candidate
+        /// </summary>
+        /// <param name="newCandidate">object of the NewCandidateViewModel class
+        /// containing information about the new candidate</param>
+        /// <returns>IActionResult</returns>
         [HttpPost]
+        [Route("Candidate/Create/{newCandidate.id}")]
```

Comment:
There's no newCandidate object at the moment. And even if it was here, you want to create something so it doesn't have id